### ISSUE TITLE
Next Step Microgrant: NFTCONSTRUCTOR.ETH

### ISSUE THREAD

**Issue by nijoe1** [filecoin-project/devgrants#830]
### 1. What is your project? 

We are a one stop Platform for artists to create NFT Collections and monetize them from our dashboard the vision is to include basic NFTs , tableland based NFTs and also some cool interactive audio visualizers.Every NFT will be stored through NFT.STORAGE. Also a future feature will be to create a Decentralized Social Layer for Projects ( NFT Collections - DAOS - ERC20 Tokens ) With projectCreators Profiles - Project Posts - and Contributions by the mainUserProfiles!

### 2. Project links
  
Link to Github repo:
Link to demo or website, if applicable:

Tableland nft constructor : 

https://github.com/sumukhah/eth_global_hackathon

Visual and audio NFTs prototype : 

https://github.com/nijoe1/_Visual-AudioNFTs_

  
### 3. How is IPFS, Filecoin, or related technology used in this project? 

 All the data are stored into Ipfs and filecoin through NFTSTORAGE we are going to use spheron to host our platform.
  
### 4. How will you improve your project with this grant? What steps will you take to meet this objective? (max 200 words)

I want to include some DAO NFT utilities using tableland and IPFS content for proposals also have some plans to encrypt some data using lit protocol. Maybe i will add some orbis chat functionalities using ceramic DIDs and lit protocol to encrypt crytical chat posts contributions information!

### 5. Do you agree to share grant reports upon request, including a final grant report at the end of the three month period?

Yes i agree. I will submit the final report at the end of the submition period. I will also include a spheron link linked with my ENS => NFTCONSTRUCTOR.ETH. 

  
### 6. Does your proposal comply with our Community Code of Conduct?
 YES 

### 7. Links and submissions
* If your project began at a hackathon, have you submitted it for the relevant Protocol Labs prizes? Include links here if available:

Tableland nft constructor : 

https://github.com/sumukhah/eth_global_hackathon

Visual and audio NFTs prototype : 

https://github.com/nijoe1/_Visual-AudioNFTs_
  
### Additional questions:
* For each team member(s), please list name, email, Github account, and role in the project.
  I will add my team members later this weak.
* How did you learn about our microgrant program?
 I just submit the corresponding Form from my prev Metabolism Hack and they reached me to contribute
* If your project was created as part of an event or hackathon:
  * What was the name of the event? 
      Metabolism-HackFS from ETH GLOBAL
  * Please link to your hackathon submission
HACKFS showcase => https://ethglobal.com/showcase/nftconstructor-dcah3
Metabolism showcase => https://ethglobal.com/showcase/visual-and-audionfts-b0r4h

**Comment by ErinOCon** [comment]
Hi @nijoe1, thank you for your proposal. Can you provide the following information?

1) Please complete items 4 and 5 of the proposal
2) Is this project open source?
3) Can you provide a working link for your prototype?

**Comment by ErinOCon** [comment]
Hi @nijoe1, I hope you are doing well! Do you expect to have an update available this week?

**Comment by nijoe1** [comment]
Hey there yeah sure  I just finished a Hackathon sorry about the delay I will update it today !

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nijoe1** [comment]
> Hi @nijoe1, thank you for your proposal. Can you provide the following information?
> 
> 1. Please complete items 4 and 5 of the proposal
Done
> 2. Is this project open source?
It will be in some weeks. Right now trying to find me team mates and working on the Conceptual part to find the exact services that we are going to provide.
> 3. Can you provide a working link for your prototype?
When the project development will begin i will provide the Spheron or Fleek link linked on my ENS nftcontructor.eth

**Comment by ErinOCon** [comment]
Hi @nijoe1, thank you for the additional information. Once this project is ready to open source, we can proceed with the grant! Please feel welcome to notify our team when this is the case.

**Comment by nijoe1** [comment]
Ok thanks I will inform u currently I am just conceptualise a scalable architecture !

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nijoe1** [comment]
Hey @ErinOCon, the project is more than ready from what we thought! Can we proceed with the microgrant?  we changed the project's name to the crypto studio and now support a lot of stuff like voting and encrypting posts based on different conditions with the lit protocol. Each collection has a decentralized discord powered by Orbis.club - ceramic network. Check  the
Dapp => https://the-crypto-studio.vercel.app/

Here is also some research for our competitors on how we are more robust and our next steps. It is a .ppt file => https://bafybeiejcbo7uw22o2ajkofsxbw2n7x73defkljllqbu7d3b4w56z6cqvq.nftstorage.web.link/ 

We also have plans to proceed with protocols live Buddle-01 for community videos and many more! 
Thanks a lot
Nick Lionis

**Comment by ErinOCon** [comment]
Hi @nijoe1, thank you for the recent email. We would be happy to complete a new review of this proposal. Can you confirm if this project complies with our [open source policy](https://github.com/filecoin-project/devgrants/blob/master/open-source-policy.md)? Also, it looks like one of the repos provided is not working.

**Comment by nijoe1** [comment]
Hey, @ErinOCon, I made a conflict. I thought that the grant was another one. That's why I removed what I have written here. However, the other grant is based on the same idea and even better. Here is the grant proposal = https://github.com/ipfs/devgrants/issues/272 , Where we include everything needed.  Sorry about the confusion. What is better to do? Bring all the information from the other grant here and delete it OR wait for a review of the other grant and delete this one?
